ui/app/AppLayouts/Wallet/views/walletconnect/sdk/src/index.js

```diff
@@ -45,10 +49,10 @@ window.wc = {
                 },
             });
 
-            window.wc.authClient = await AuthClient.init({
-                projectId: projectId,
-                metadata: window.wc.web3wallet.metadata,
-            });
+            //window.wc.authClient = await AuthClient.init({
```

Comment:
@stefandunca why did you disable this?

Reply:
I did it while debugging the issue with different pairing topics between the client and the wallet. It is still to be turned on when the auth is appropriately implemented.